app/Providers/AppServiceProvider.php

```diff
@@ -375,6 +380,29 @@ public function boot()
             return true;
         });
 
+        Validator::extend('lead_report_settings_dto_array', function ($attribute, $value, $parameters, $validator) {
+            $validator->addReplacer('lead_report_settings_dto_array', function ($message, $attribute, $rule, $parameters) use ($validator) {
+                return sprintf
+                (
+                    "%s should be an array of strings or extra question data {id : int, name: string}",
+                    $attribute);
+            });
+
+            if (!is_array($value)) return false;
+
+            foreach ($value as $element) {
+                if (is_string($element)) continue;
+
+                foreach ($element as $key => $element_val) {
```

Comment:
this validation will not work when this test https://github.com/OpenStackweb/summit-api/pull/239/files#r1539902496 is corrected